### PR TITLE
Implemented starting of the Game.

### PR DESCRIPTION
**Pull request by Ranray**

### CODE REVIEW

File: Soccar [Game-Server]/src/nl/soccar/gameserver/SessionController.java [hunk 1]
```diff
@@ -152,6 +153,18 @@ public void leaveSession(Player player, Session session) {
         TeamColour colour = teamBlue.getPlayers().stream().filter(p -> p.getUsername().equals(player.getUsername())).count() > 0 ? TeamColour.BLUE : TeamColour.RED;
         leaveSession(player, colour, session);
     }
+    
+    public void startGame(Player player, Session session) {
```
Comment (TehMessiah) [inline Soccar [Game-Server]/src/nl/soccar/gameserver/SessionController.java]: MAJOR Remove this unused method parameter "player". [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1172)
MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 2 issues
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

File: Soccar [Game-Server]/src/nl/soccar/gameserver/SessionController.java [hunk 2]
```diff
@@ -152,6 +153,18 @@ public void leaveSession(Player player, Session session) {
         TeamColour colour = teamBlue.getPlayers().stream().filter(p -> p.getUsername().equals(player.getUsername())).count() > 0 ? TeamColour.BLUE : TeamColour.RED;
         leaveSession(player, colour, session);
     }
+    
+    public void startGame(Session session) {
```
Comment (ghost) [inline Soccar [Game-Server]/src/nl/soccar/gameserver/SessionController.java:157]: MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Comment by ghost** [comment]
SonarQube analysis reported 1 issue
* MINOR 1 minor

Watch the comments in this conversation to review them.

File: Soccar [Game-Server]/src/nl/soccar/gameserver/message/handler/StartGameMessageHandler.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+           connection.close();
+           return;
+       }
+       
+       Session session = player.getCurrentSession();
+       if (session == null) {
+           connection.close();
+           return;
+       }
+        SessionController.getInstance().startGame(session);
```
Comment (BeeHiveJava) [inline Soccar [Game-Server]/src/nl/soccar/gameserver/message/handler/StartGameMessageHandler.java:31]: Vage indentatie + whitespace.

**Review by BeeHiveJava — CHANGES_REQUESTED** [review]

**Review by BeeHiveJava — APPROVED** [review]

**Pull request merged**